mono/mini/Makefile.am.in

```diff
@@ -262,8 +262,12 @@ mono_sgen_LDADD = \
 	-lm			\
 	$(MONO_DTRACE_OBJECT)
 
-mono_sgen_LDFLAGS = $(static_flags) $(monobinldflags) $(monobin_platform_ldflags) 
 
+mono_sgen_LDFLAGS = $(STATIC_GCC_LIBS_CFLAGS) $(static_flags) $(monobinldflags) $(monobin_platform_ldflags) 
```

Comment:
We did similar in a different project in order to get full static linking, but instead of having all the flags, we just used 
-all-static flag. That produces a mingw build cross compiler that doesn't have any dependecies on c/c++ runtimes.

Reply:
In that case we only used that flag together with the -all-static when linking mono runtime and that produced a cross compiler without any additional dependecies.